src/sonic-config-engine/tests/test_j2files.py

```diff
@@ -323,6 +323,42 @@ def test_qos_dscp_remapping_render_template(self):
             assert utils.cmp(sample_output_file, test_output)
             os.remove(test_output)
 
+    def test_config_brcm_render_template(self):
+        if utils.PYvX_DIR != 'py3':
+            #Skip on python2 as the change will not be backported to previous version
+            return
+
+        dir_paths = [
+            '../../../device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-D48C8',
```

Comment:
i suggest to get the config.bcm local to the tests folder, to make the package self-contained.

Reply:
yes, the j2 generated file should be there, added.

Reply:
if we place a replica of the j2 template into test folder, then we need to keep them sync manully when they changed?